src/operators/suffix.ts

```diff
@@ -30,6 +30,13 @@ export enum Suffixes {
   Reals = '_reals',
 }
 
+// squirrel the suffix values away so we can use it as a lookup later
+const SuffixValueLookup = new Map<string, string>();
```

Comment:
We could also use an `Object` as a `Record<string, string>` (or `Record<string, boolean>` since the value isn't significant). I don't have a strong argument for changing this away from a `Map`, though. If we used an object, a common pattern would be to reduce the `Object.values` e.g.:

```
const suffixValueLookup = Object.values(Suffixes).reduce<Record<string, boolean>>(
    (acc, val) => ({
      ...acc,
      [val]: true
    }),
    {}
  );
```

No change required here unless you prefer this approach.

Reply:
I went to change it to this, but I also lost the has method, which seems like it became less readable.  I did get rid of a temporary variable to look more like this but left the Map.  Researched online and it didn't seem to be a major difference in performance